### PR TITLE
Third person readme

### PR DESCRIPTION
**Pull request by tTakeda09**
**Sample**

Third person character

**Summary**

- Add the How to use sample section
- Make the How it works section aligned with the actual sample.

**Tests**

Read it

**ArcGIS Maps SDK Version**

### CODE REVIEW

File: sample_project/Content/SampleViewer/Samples/ThirdPersonCharacter/readme.md [hunk 1]
```diff
@@ -4,13 +4,24 @@ Allows users to explore a city/map from the perspective of a Third Person Charac
...
 ## How it works
 
-1. Check the box for **Mesh Colliers Enabled** on the `ArcGIS Map` component.
-2. Create a parent object with an `ArcGIS Location` component.
-3. Create child game objects in the scene that use `Transform` components.
-4. Using the `ArcGIS Map`, change the latitude and longitude of where you would like to explore.
-5. Add your third person character controller to the level and position it slightly above the ground.
+1. Set up **ArcGIS Map** for the exploration area.
+2. Create a Third Person Character with its control and attach the [**ArcGIS Camera Component**](https://developers.arcgis.com/unreal-engine/maps/camera/#arcgis-camera-component) to the Character Mesh. 
+3. Add the Third Person Character to the level.
```
Comment (Jade-JadeH) [inline sample_project/Content/SampleViewer/Samples/ThirdPersonCharacter/readme.md:21]: which one? the built in or the sample one?

Comment (tTakeda09) [reply sample_project/Content/SampleViewer/Samples/ThirdPersonCharacter/readme.md:21]: I'm expecting the built-in one but after doing the 2...

Comment (tTakeda09) [reply sample_project/Content/SampleViewer/Samples/ThirdPersonCharacter/readme.md:21]: What do you mean by `it didn't work...`? Can you explain what was going on?
I tried a quick test using the Third Person template by deleting blocked etc from the template label and adding ArcGIS Map + Adding ArcGIS Camera to the Third Person character, the map was loaded and the character could move around.
Note, I understand that we don't need to explain how to use UE and only focus on what should be done to use our plugin. Clearly, my third-person test is dirty, and another camera comes together with the template I didn't remove (need to modify in C++). The third-person character in our sample is cleaner and all done by BP. But that's one of the ways users can create a third-person character and the only key point here is attaching the ArcGIS Camera to the character.

Comment (Jade-JadeH) [reply sample_project/Content/SampleViewer/Samples/ThirdPersonCharacter/readme.md:21]: ok, I might missed some steps you did there. Sounds good then!

**Review by Jade-JadeH — APPROVED** [review]

**Comment by mrnelson6** [comment]
Why change the image size?

File: sample_project/Content/SampleViewer/Samples/ThirdPersonCharacter/readme.md [hunk 2]
```diff
@@ -1,16 +1,27 @@
-# Third Person Character Controller
+# Explore with Third Person Character
```
Comment (tTakeda09) [inline sample_project/Content/SampleViewer/Samples/ThirdPersonCharacter/readme.md:1]: @Jade-JadeH @avr757 I missed changing the title here. What do you think?

**Comment by tTakeda09** [comment]
> Why change the image size?

To reduce the build time. It takes a longer time to be built if the image file is big and it's painful when we build the site locally.

File: sample_project/Content/SampleViewer/Samples/ThirdPersonCharacter/readme.md (hunk 2 above)
Comment (Jade-JadeH) [reply sample_project/Content/SampleViewer/Samples/ThirdPersonCharacter/readme.md:1]: I think this change looks good

**Pull request merged**